src/core/web_api/p2_api.js

```diff
@@ -17,11 +18,24 @@ class Auth0APIClient {
     this.isUniversalLogin = window.location.host === domain;
     this._enableIdPInitiatedLogin = !!(opts._enableIdPInitiatedLogin || opts._enableImpersonation);
 
-    const default_telemetry = {
-      name: 'lock.js',
-      version: __VERSION__,
-      lib_version: auth0.version
-    };
+    const { auth0Client } = qs.parse(window.location.search.substr(1));
```

Comment:
won't this include anything on the query, not just the auth0Client query param?

Reply:
this is parsing the query string and getting only the `auth0Client` key

Reply:
Ah, I just see the destructuring 👍